device/samples/typescript/readme.md

```diff
@@ -0,0 +1,123 @@
+# Typscript Samples for the Azure IoT device SDK for Node.js
```

Comment:
question: how do you feel about the maintainability of the two separate readmes? I'm concerned since they're copy paste that one will get out of alignment over time. Perhaps put the readme in the samples folder directly and have small readmes for how to run each sample type, i.e. javascript vs. typescript?

Reply:
this works for me, i will adjust